Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -198,6 +198,8 @@ It is important to remember that Local Authentication framework is an event-base
 - Verify that sensitive processes, such as re-authenticating a user triggering a payment transaction, are protected using the Keychain services method.
 - Verify that the `kSecAccessControlTouchIDAny` or `kSecAccessControlTouchIDCurrentSet` flags are set and `kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly` protection classes are set when the `SecAccessControlCreateWithFlags` method is called. Note that, alternatively, `kSecAccessControlUserPresence` can be used as a flag as well when you want to be able to use passcode as a fallback. Last, note that, when `kSecAccessControlTouchIDCurrentSet` is set, changing the fingerprints registered to the device will invalidate the entry which is protected with the flag.
 
+Please note that using `kSecAccessControlTouchIDAny` will allow the keymaterial to be used when the user (re-)enroles new fingerprints, but it will mean that attackers who are somehow able to enrole their fingerprints to the device, now can access items that use this flag as well.
```

Comment:
Keep it consistent by referring to "entries". Split and made the sentences shorter.

```suggestion
Please note that using `kSecAccessControlTouchIDAny` will allow the entry to be used when the user (re-)enroles new fingerprints. However, it means also that attackers, who are somehow able to enrole their fingerprints to the device, can now access those entries as well.
```

Reply:
Modified your comment for a grammar improvement since I can't propose a new commit suggestion on top of yours.